nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -2,7 +2,7 @@ collision_monitor:
   ros__parameters:
     base_frame_id: "base_footprint"
     odom_frame_id: "odom"
-    cmd_vel_in_topic: "cmd_vel_raw"
+    cmd_vel_in_topic: "cmd_vel_smoothed"
```

Comment:
What about for other param files (e.g. any other yaml files in this package from the collision detection node merge recently? + the yaml files in the `nav2_bringup` package)

Reply:
@SteveMacenski I couldn't find any other place setting this param in the codebase, or any yaml file setting the collision monitor params. There is the test file `navigation2/nav2_collision_monitor/test/collision_monitor_node_test.cpp`, but I think the value currently there `cmd_vel_in` is the expected one. .